src/Generator/GeizhalsDE.php

```diff
@@ -100,25 +93,15 @@ protected function generatePluginContent($elasticSearch, array $formatSettings =
 
             do
             {
-                $this->getLogger(__METHOD__)->debug('ElasticExportGeizhalsDE::logs.writtenLines', [
-                    'Lines written' => $limit,
-                ]);
-
                 if($limitReached === true)
                 {
                     break;
                 }
 
-                $esStartTime = microtime(true);
-
                 // Get the data from Elastic Search
                 $resultList = $elasticSearch->execute();
 
-                $this->getLogger(__METHOD__)->debug('ElasticExportGeizhalsDE::logs.esDuration', [
```

Comment:
It's a bit redundant because the entire process duration is counted also the Items class that started this process. So we agreed with Max that we will make use just of that log and remove this ones.